include/constants.h

```diff
@@ -194,12 +210,13 @@ enum icontrols
     /*! \brief Divide the intervals of the grid by 2^(divideBy2)
          *
          * Divides the intervals of the grid by 2^(divideBy2)
+         * (defines how often to split the intervals of the grid at the midpoint)
          */
     divideBy2,
     /*! \brief Problem to solve
          *
          * Defines the problem to solve
-         * - 5: the classical problem (Zoni2019, aligned with the cartesian grid)
+         * - 5: the classical problem (Zoni2019, aligned with the cartesian grid) (we always use this)
```

Comment:
```suggestion
         * - 5: the classical problem (Zoni2019, aligned with the cartesian grid)
```

The latter is not correct